hazelcast/src/main/java/com/hazelcast/kubernetes/KubernetesClient.java

```diff
@@ -706,27 +738,28 @@ final class StsMonitor implements Runnable {
          */
         @Override
         public void run() {
-            String stsUrlString = String.format("%s/apis/apps/v1/namespaces/%s/statefulsets", kubernetesMaster,
-                    namespace);
-            JsonObject jsonObject = callGet(stsUrlString);
-            latestResourceVersion = jsonObject.get("metadata").asObject().getString("resourceVersion",
-                    null);
-            latestRuntimeContext = extractStsList(jsonObject);
-            LOGGER.info("Initializing cluster topology tracker with initial context: "
-                    + latestRuntimeContext);
-            clusterTopologyIntentTracker.update(UNKNOWN,
-                    latestRuntimeContext.getSpecifiedReplicaCount(),
-                    UNKNOWN, latestRuntimeContext.getReadyReplicas(),
-                    UNKNOWN, latestRuntimeContext.getCurrentReplicas());
-            while (true) {
+            BackoffIdleStrategy backoffIdleStrategy = new BackoffIdleStrategy(3, MAX_YIELDS,
```

Comment:
Can we move 3 to MAX_SPINS constant?

Reply:
extracted all backoff idle strategy constants in 77a8077a399212e80268a99ff5d4e9263a4d35b4